tools/functional-tester/etcd-tester/etcd_runner_stresser.go

```diff
@@ -0,0 +1,64 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os/exec"
+
+	"syscall"
+)
+
+type runnerStresser struct {
+	cmd    *exec.Cmd
+	cmdStr string
+	args   []string
+
+	errorc chan error
+}
+
+func (rs *runnerStresser) setupOnce() (err error) {
+	if rs.cmdStr == "" {
+		return fmt.Errorf("expect cmdStr to be set")
+	}
+	if rs.cmd != nil {
+		return nil
+	}
+
+	rs.cmd = exec.Command(rs.cmdStr, rs.args...)
+	stderr, err := rs.cmd.StderrPipe()
+
+	go func() {
+		out, err := ioutil.ReadAll(stderr)
```

Comment:
will etcd-runner exit immediately after writing to stderr or is it possible this will keep reading never data from stderr and the runner never quits?